sklearn/ensemble/tests/test_forest.py

```diff
@@ -1337,25 +1337,3 @@ def test_backend_respected():
         clf.predict_proba(X)
 
     assert ba.count == 0
-
-
-@pytest.mark.filterwarnings('ignore:The default value of n_estimators')
-@pytest.mark.parametrize('name', FOREST_CLASSIFIERS)
-@pytest.mark.parametrize('oob_score', (True, False))
-def test_multi_target(name, oob_score):
-    ForestClassifier = FOREST_CLASSIFIERS[name]
-
-    clf = ForestClassifier(bootstrap=True, oob_score=oob_score)
-
-    X = iris.data
-
-    # Make multi column mixed type target.
-    y = np.vstack([
-        iris.target.astype(float),
-        iris.target.astype(int),
-        iris.target.astype(str),
```

Comment:
The original test, uses y with dtype `<U32` which contains strings. The new common test only accounts for numeric types.

Reply:
Good point. I added that check again for estimators that don't use LabelBinarizer.